lib/redis/hash_key.rb

```diff
@@ -130,7 +130,9 @@ def fill(pairs={})
     # Get keys in bulk, takes an array of fields as arguments. Redis: HMGET
     def bulk_get(*fields)
       hsh = {}
-      res = redis.hmget(key, *fields.flatten)
+      get_fields = *fields.flatten
+      get_fields << nil if get_fields.empty?
+      res = redis.hmget(key, get_fields)
```

Comment:
could probably be a formatter method for both cases and add tests.. but basically I'm documenting this bug with a PR :)